doc/grammars/type.abnf

```diff
@@ -28,7 +28,11 @@ Atomic
 	/ TokenParenthesesOpen ParenthesizedType TokenParenthesesClose [Array]
 
 Generic
-	= TokenAngleBracketOpen Type *(TokenComma Type) TokenAngleBracketClose
+	= TokenAngleBracketOpen GenericTypeArgument *(TokenComma GenericTypeArgument) TokenAngleBracketClose
+
+GenericTypeArgument
+    = [TokenContravariant / TokenCovariant] Type
```

Comment:
nit: tabs vs. spaces

Reply:
But which one is correct? It's pretty inconsistent across this whole file 😃